README.md

```diff
@@ -18,16 +18,10 @@ It works on CPython 2.7. It is known to work on Ubuntu Linux and OS X.
 | [![Build Status](https://travis-ci.org/Netflix/security_monkey.svg?branch=develop)](https://travis-ci.org/Netflix/security_monkey)  | [![Build Status](https://travis-ci.org/Netflix/security_monkey.svg?branch=master)](https://travis-ci.org/Netflix/security_monkey)  |
 | [![Coverage Status](https://coveralls.io/repos/github/Netflix/security_monkey/badge.svg?branch=develop)](https://coveralls.io/github/Netflix/security_monkey?branch=develop)  | [![Coverage Status](https://coveralls.io/repos/github/Netflix/security_monkey/badge.svg?branch=master)](https://coveralls.io/github/Netflix/security_monkey?branch=master) |
 
-
-🚨⚠️🥁🎺 PLEASE READ: BREAKING CHANGES FOR 1.0 🎺🥁⚠️🚨
```

Comment:
Please keep this here. There are very important breaking changes for 1.0.

Reply:
Done: https://github.com/Netflix/security_monkey/pull/950/commits/51c9079f5b3080a0f78bf978725074d0dd47e38c